shared/actions/platform-specific.desktop.js

```diff
@@ -26,6 +27,17 @@ function setNoPushPermissions(): Promise<*> {
   throw new Error('Push permissions unsupported on this platform')
 }
 
+function setAppState(toMerge: Object) {
```

Comment:
this data is stored and dealt with on the node side so we pass messages across the channel to get/set it